pom.xml

```diff
@@ -121,8 +121,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>8</source>
-                    <target>8</target>
+                    <source>9</source>
```

Comment:
Why did this change? Is it relevant to this PR?

Reply:
IntelliJ prompted me to switch this after making a change earlier. I reverted this change without any issues, so this seems unnecessary.